src/main/java/com/transportcompany/transportcompany/services/OrderServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.transportcompany.transportcompany.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.transportcompany.transportcompany.exceptions.OrderNotFoundException;
+import com.transportcompany.transportcompany.models.dtos.OrderDTO;
+import com.transportcompany.transportcompany.models.entities.Order;
+import com.transportcompany.transportcompany.repositories.OrderRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public  class OrderServiceImpl implements OrderService {
```

Comment:
As a small improvement in this service (and/or in all of them), you can use a logger to log every CRUD operation. For example, when you create an order it will be useful to log that event as "User yyy was created".